ext-apiserver/cloud/google/pods.go

```diff
@@ -27,29 +29,81 @@ import (
 	"github.com/golang/glog"
 )
 
-var machineControllerImage = "gcr.io/k8s-cluster-api/machine-controller:0.19"
+var machineControllerImage = "gcr.io/k8s-cluster-api/apiserver-controller:0.1"
 
 func init() {
 	if img, ok := os.LookupEnv("MACHINE_CONTROLLER_IMAGE"); ok {
 		machineControllerImage = img
 	}
 }
 
+type caCertParams struct {
+	caBundle string
+	tlsCrt   string
+	tlsKey   string
+}
+
+func getBase64(file string) string {
+	buff := bytes.Buffer{}
+	enc := base64.NewEncoder(base64.StdEncoding, &buff)
+	data, err := ioutil.ReadFile(file)
+	if err != nil {
+		glog.Fatalf("Could not read file %s: %v", file, err)
+	}
+
+	_, err = enc.Write(data)
+	if err != nil {
+		glog.Fatalf("Could not write bytes: %v", err)
+	}
+	enc.Close()
+	return buff.String()
+}
+
+func getApiServerCerts() (*caCertParams, error) {
+	os.RemoveAll("./config")
+	err := run("apiserver-boot", "build", "config",
+		"--name", "mycontroller", "--namespace", "default",
+		"--image", "gcr.io/fmin-gke-dev/controller:0.2")
+	if err != nil {
+		return nil, err
+	}
+
+	certParms := &caCertParams{
+		caBundle: getBase64("config/certificates/apiserver_ca.crt"),
+		tlsCrt:   getBase64("config/certificates/apiserver.crt"),
+		tlsKey:   getBase64("config/certificates/apiserver.key"),
+	}
+
+	return certParms, nil
+}
+
 func CreateMachineControllerPod(token string) error {
-	tmpl, err := template.ParseFiles("cloud/google/pods/machine-controller.yaml")
+	tmpl, err := template.ParseFiles("cloud/google/pods/apiserver.yaml")
```

Comment:
This is no longer a pod. It's a huge bundle. We should change the location of this file, this template name, and the medtho `createPod` as it's no longer creating a pod.

Reply:
Done